app/fixtures/ember/canary.js

```diff
@@ -7,5 +7,5 @@ export default {
   enableTestURL: true,
   debugFileName: ".debug.js",
   ignoreFiles: ['ember.js'],
-  installWithEmberCLI: '# Install the latest Ember canary:\nbower install --save ember#canary'
+  installWithEmberCLI: '# Install the latest Ember canary:\nnpm install --save-dev emberjs/ember.js#master'
```

Comment:
This command won't actually work (as it would not have the precompiled dist assets).

Reply:
@rwjblue So we cannot install it via npm.